### ISSUE TITLE
unable to handle batchnorm1d in pytorch

### ISSUE THREAD

**Issue by lonelykid** [waleedka/hiddenlayer#50]
A simple example, no issues
![image](https://user-images.githubusercontent.com/2936633/61998963-c55f3200-b085-11e9-9d16-79f1935cd4f0.png)
When added batchnorm1d between layers, 
```
Expected more than 1 value per channel when training, got input size torch.Size([1, 100])
```
It seems like it's treating batchnorm1d as batchnorm2d?
